### ISSUE TITLE
 normalize for GRNN

### ISSUE THREAD

**Issue by wewill7** [itdxer/neupy#269]
I want to know if the library has implemented the normalize for my input data? because when I change the 'std', I find it is very little  is also ok despite my data is large. It troubles me . But I did not find the code that  implements the normalization.

**Comment by ahmadjordan** [comment]
In the example on Neupy page they do normalization from the preprocessing in  Skitlearn library:

import numpy as np
from sklearn import datasets, preprocessing
from sklearn.model_selection import train_test_split
from neupy import algorithms

**dataset = datasets.load_diabetes()
x_train, x_test, y_train, y_test = train_test_split(
    preprocessing.minmax_scale(dataset.data),
    preprocessing.minmax_scale(dataset.target.reshape(-1, 1)),
    test_size=0.3,
)**

nw = algorithms.GRNN(std=0.1, verbose=False)
nw.train(x_train, y_train)

y_predicted = nw.predict(x_test)
mse = np.mean((y_predicted - y_test) ** 2)
ms
